pallets/capacity/src/lib.rs

```diff
@@ -195,6 +195,18 @@ pub mod pallet {
 	#[pallet::getter(fn get_current_epoch_used_capacity)]
 	pub type CurrentEpochUsedCapacity<T: Config> = StorageValue<_, BalanceOf<T>, ValueQuery>;
 
+	#[pallet::type_value]
+	/// EpochLength defaults to T::MaxEpochLength when not set
+	pub fn EpochLengthDefault<T: Config>() -> T::BlockNumber {
+		T::MaxEpochLength::get()
+	}
+
+	/// Storage for the epoch length
+	#[pallet::storage]
+	#[pallet::getter(fn get_epoch_length)]
+	pub type EpochLength<T: Config> =
+		StorageValue<_, T::BlockNumber, ValueQuery, EpochLengthDefault<T>>;
```

Comment:
whats the default?

Reply:
It defaults to `MaxEpochLength` set in the config (if there is no value in storage)